### PR TITLE
Break up files with multiple classes

### PR DESCRIPTION
**Pull request by e7mac**
Breaking apart files with multiple classes

Renaming tests with the class they are testing

### CODE REVIEW

File: Leanplum-SDK/Classes/LPResponse.m [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+#import "Leanplum.h"
+#import "LeanplumInternal.h"
+#import "LeanplumRequest.h"
+#import "LPResponse.h"
+#import "Constants.h"
+#import "LPFileManager.h"
+#import "NSTimer+Blocks.h"
+#import "LPKeychainWrapper.h"
+#import "LPEventDataManager.h"
+#import "LPEventCallbackManager.h"
```
Comment (alexisoyama) [inline Leanplum-SDK/Classes/LPResponse.m]: Let's clean this part up too

File: Leanplum-SDK/Classes/LPResponse.h [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+//  Unless required by applicable law or agreed to in writing,
+//  software distributed under the License is distributed on an
+//  "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+//  KIND, either express or implied.  See the License for the
+//  specific language governing permissions and limitations
+//  under the License.
+
+#import <Foundation/Foundation.h>
+#import "Leanplum.h"
+#import "LPNetworkFactory.h"
```
Comment (alexisoyama) [inline Leanplum-SDK/Classes/LPResponse.h]: remove ?

File: Leanplum-SDK/Classes/LPInternalState.h [hunk 3]
```diff
@@ -9,8 +9,6 @@
 #import "LPRegisterDevice.h"
 #import "LPActionManager.h"
 
-#pragma mark - LPInternalState interface
```
Comment (alexisoyama) [inline Leanplum-SDK/Classes/LPInternalState.h]: I don't mind removing these but note that these are here to make it look cleaner on quick function view.

**Comment by alexisoyama** [comment]
also let's have a better naming for the title ;)

File: Leanplum-SDK/Classes/LPResponse.m (hunk 1 above)
Comment (e7mac) [reply Leanplum-SDK/Classes/LPResponse.m]: I'm gonna do another pass for cleaning up imports

**Pull request merged**